Services/DataBaseService/BrokerConsumers/CreateUserConsumer.cs

```diff
@@ -19,7 +19,7 @@ public CreateUserConsumer([FromServices] IUserRepository userRepository)
         private OperationResult CreateUser(InternalCreateUserRequest request)
         {
             userRepository.CreateUser(request.User);
-            userRepository.CreateUserCredential(request.Credential);
+            userRepository.CreateUserCredential(request.Credential);            

```

Comment:
What is this space? 
Install this [extension](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer), it's cool

Reply:
Thanks for link)